DirectX11Proj/IScene.h

```diff
@@ -17,19 +17,19 @@ class IScene
 	IScene();
 	~IScene();
 
-	virtual void Tick(InputClass* const aInput) = 0;
+	virtual void Tick(InputClass* const apInput) = 0;
 	virtual void Init() = 0;
 	virtual void Destroy() = 0;
 
 	// Just a bit lazy and turn it into a vector, will convert back to a list later
-	std::vector <std::shared_ptr<IObject>> mObjects;
+	std::vector <std::unique_ptr<IObject>> mObjects;
```

Comment:
http://stackoverflow.com/questions/238008/relative-performance-of-stdvector-vs-stdlist-vs-stdslist

Lists has its uses but this question and the people answering elaborates a bit more on my decisions